app/views/posts/index.html.erb

```diff
@@ -23,23 +25,30 @@
         <% end %>
       </div>
     </div>
-    <div>
-      <h3>Comments</h3>
-      <% post.comments.each do |comment| %>
-        <p> <%= comment.user.name %> said: <%= comment.body %></p>
-      <% end %>
-    </div>
-    <div>
-      Add a comment:
-      <%= form_for([post, post.comments.build]) do |comment| %>
-        <%= comment.hidden_field(:user_id, value: current_user.id) %>
-        <p> <%= comment.text_area :body %></p>
-        <p> <%= comment.submit %> </p>
-      <% end %>
+
+    <!-- Another div for comment form and comments  -->
+    <div class='comment-container'>
+
+      <!-- Comments are listed in this part of the div -->
+      <div class='comment-list-item'>
+        <% post.comments.each do |comment| %>
+          <p> <span class='comment-user-name'> <%= comment.user.name %>: </span><%= comment.body %></p>
+        <% end %>
+      </div>
+
+      <!-- Here is where the comment form lives -->
+      <div class='comment-form'>
+        <div class='comment-col-left float-left'>
+          <%= form_for([post, post.comments.build]) do |comment| %>
+            <%= comment.hidden_field(:user_id, value: current_user.id) %>
+            <%= comment.text_field :body, placeholder: 'Write a comment... ✏️', class: 'comment-input-box float-left' %>
+            <%= comment.submit class: 'hidden' %>
+          <% end %>
+        </div>
+      </div>
```

Comment:
Just because it's a learning opportunity I haven't mentioned much of — look up Rails partials as a way of breaking up templates (will help us avoid the comments).